developer_guide/installation/installation_workstation.rst

```diff
@@ -375,6 +375,33 @@ Adding the vhost name
     $ sudo gedit /etc/hosts
     127.0.0.1    akeneo-pim.local
 
+Configuring tasks via crontab
+*****************************
+
+The application needs the following tasks to be executed in background on a regular basis :
+
+.. code-block:: bash
+    :linenos:
+
+    # for CE & EE
+    /path/to/php /path/to/pim/root/app/console pim:completeness:calculate
+    /path/to/php /path/to/pim/root/app/console pim:versioning:refresh
+
+    # for EE only
+    /path/to/php /path/to/pim/root/app/console akeneo:rule:run
+
+Edit your crontab with `crontab -e` and configure each task. For example, the following line will run the completeness calculation every 15 minutes :
+
+.. code-block:: bash
+    :linenos:
+
+    # m  h  dom  mon  dow  command
+    */15 *  *    *    *    /path/to/php /path/to/pim/root/app/console pim:completeness:calculate > /path/to/pim/root/app/logs/calculate_completeness.log 2>&1
```

Comment:
Maybe add a note about 2>&1 to say it redirect stderr to stdout wich is the log file